wolvic/browser/vr/moz_external_vr.h

```diff
@@ -429,6 +428,11 @@ struct VRControllerState {
   // https://immersive-web.github.io/webxr/#dom-xrinputsource-targetrayspace
   VRPose targetRayPose;
 
+  // When Cap_Orientation is set in flags, targetRayOrientation corresponds
+  // to the orientation of the target ray (origin of the ray is determined by
+  // the controller position). Only used by the Chromium backend.
+  float targetRayOrientation[4];
```

Comment:
As mentioned I don't think this is the right approach. We should already have all the available data because we can do it with Gecko. Have you tried to mimic Gecko's code to get the info you need?